docs/source/api/distributions/transforms.rst

```diff
@@ -24,7 +24,6 @@ implemented as ``pm.distributions.transforms.``\*X*.
    ordered
    log
    sum_to_1
-   t_stick_breaking
```

Comment:
removed in https://github.com/pymc-devs/pymc3/pull/4129 , looks like it was never removed from here though